File: app/src/main/java/com/rrdev/roombookingmvvm/data/Firebase/MyFirebaseInstanceIDService.java
//package com.rrdev.roombookingmvvm.data.Firebase;
//
//import android.util.Log;
//import com.google.firebase.iid.FirebaseInstanceIdService;
//import com.google.firebase.iid.FirebaseInstanceId;
//import com.rrdev.roombookingmvvm.data.SharedPreferences.SharedPrefToken;
//
//public class MyFirebaseInstanceIDService extends FirebaseInstanceIdService {
//
//    private static final String TAG = "MyFirebaseIIDService";
//    @Override
//    public void onTokenRefresh() {
//        String refreshedToken = FirebaseInstanceId.getInstance().getToken();
//        Log.d(TAG, "Refreshed token: " + refreshedToken);
//        storeToken(refreshedToken);
//    }
//
//    private void storeToken(String token) {
//        //saving the token on shared preferences
//        SharedPrefToken.getInstance(getApplicationContext()).saveDeviceToken(token);
//    }
//}

File: app/src/main/java/com/rrdev/roombookingmvvm/data/Firebase/MyFirebaseInstanceIDService.kt
package com.rrdev.roombookingmvvm.data.Firebase

import com.google.firebase.messaging.FirebaseMessagingService
import com.google.firebase.messaging.RemoteMessage

class MyFirebaseInstanceIDService: FirebaseMessagingService() {
    override fun onMessageReceived(p0: RemoteMessage) {
        super.onMessageReceived(p0)
    }

    override fun onNewToken(token: String) {

    }


}